### PR TITLE
Added Wishlist Management Page

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added the Wishlist Management Page to manage the items in the Wishlist.

Here is the Preview Link : 
https://deploy-preview-3--distracted-wilson-c25ae0.netlify.app/wishlist/wishlist

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+:root{
+    --btn-hover-color:#6666f3;
+}
+.wishlist-wrapper{
+    margin-top: 15vh;
+}
+.wishlist-container-center{
+    display: flex;
+    flex-direction: column;
```
Comment (tusharkandpal) [inline wishlist/wishlist.css:9]: this is causing your products to show in column. instead should be in row

**Review by tusharkandpal — APPROVED** [review]
just one suggestion below. then you can merge! 👍🏻

File: wishlist/wishlist.css (hunk 1 above)
Comment (atharvabhanage02) [reply wishlist/wishlist.css:9]: I wanted to them to show up in column .

**Review by Kothalkarpriya — APPROVED** [review]
Good Work! You can merge

**Pull request merged**